<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeSelection</class>
 <widget class="QWidget" name="TimeSelection">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1092</width>
    <height>659</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TimeSelection</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Channel Traces</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="update_fig">
          <property name="text">
           <string>Update Figure</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="QSpinBox" name="left_ch">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="left_scroll">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QWidget" name="fig" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QSpinBox" name="right_ch">
              <property name="minimum">
               <number>1</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSlider" name="right_scroll">
              <property name="minimum">
               <number>1</number>
              </property>
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="slider_start">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slider_end">
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>1000</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>0</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Additional Controls</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>time start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="start_time"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>time end</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QDoubleSpinBox" name="end_time"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Selected Channels</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Dropped Channels</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QListView" name="select_list"/>
          </item>
          <item>
           <widget class="QListView" name="drop_list">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ok">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
